### ISSUE TITLE
Avoid duplication of build disk images for same image 

### ISSUE THREAD

**Issue by veniceofcode** [containers/podman-desktop-extension-bootc#385]
### Is your enhancement related to a problem? Please describe

I think it will be helpful to make users aware if they are repeating a task that is already in progress because of a previous action

### Describe the solution you'd like

Add a warning message while trying to build the same disk image more than once

**Comment by cdrage** [comment]
> ### Is your enhancement related to a problem? Please describe
> I think it will be helpful to make users aware if they are repeating a task that is already in progress because of a previous action
> 
> ### Describe the solution you'd like
> Add a warning message while trying to build the same disk image more than once
> 
> ### Describe alternatives you've considered
> _No response_
> 
> ### Additional context
> _No response_

You are right, this is not implemented, there are no checks and you can re-run the same image build against the current one. Possibly creating a conflicting build process / write path (not to mention duplicate logs).

We'll work on implementing this 👍 thank you for the awesome feedback @veniceofcode

**Comment by cdrage** [comment]
Closing for now as we have the "are you sure you want to override" option if building another disk image that's in the same output directory, preventing "duplications".
